src/modules/cp/components/ApproveRequestBlade/ApproveRequestBlade.tsx

```diff
@@ -55,11 +72,25 @@ const ApproveRequestBlade: FC<ApproveRequestBladeProps> = (props) => {
 	} = useForm<ApproveRequestFormState>({
 		resolver: yupResolver(APPROVE_REQUEST_FORM_SCHEMA()),
 		defaultValues: {
-			accessFrom: defaultAccessFrom(new Date()),
-			accessTo: defaultAccessTo(new Date()),
+			accessFrom: asDate(selected?.startAt) || defaultAccessFrom(new Date()),
+			accessTo: asDate(selected?.endAt) || defaultAccessTo(new Date()),
 		},
 	});
 
+	useEffect(() => {
+		if (selected) {
+			if (selected.startAt) {
+				setValue('accessFrom', asDate(selected.startAt));
+			}
+			if (selected.endAt) {
+				setValue('accessTo', asDate(selected.endAt));
+			}
+			if (selected.note) {
+				setValue('accessRemark', selected.note);
+			}
+		}
+	}, [selected, setValue]);
```

Comment:
Kunnen we hier geen `defaultValues` voor gebruiken?

Reply:
nee, want die blade wordt al geladen bij het laden van de pagina, maar de selectedVisitRequest is pas gekend later. en die defaultValues picken geen changes meer op na component initialisatie.

Ik had ook ff geprobeerd die blade pas te laden als een selectedVisitRequest beschikbaar is, maar dan verliezen we de slide-in animatie.